packages/react-broadcast/src/components/test-context-provider/test-context-provider.tsx

```diff
@@ -0,0 +1,52 @@
+import type {
+  TFlags,
+  TReconfigureAdapter,
+  TAdapterStatus,
+  TAdapterIdentifiers,
+} from '@flopflip/types';
+
+import React from 'react';
+import { AdapterConfigurationStatus, AdapterSubscriptionStatus } from '@flopflip/types';
+import { createAdapterContext, AdapterContext } from '@flopflip/react';
+import { createIntialFlagsContext, FlagsContext } from '../flags-context';
+
+type TProps = {
+  children: React.ReactNode;
+  flags: TFlags;
+  adapterIdentifiers?: TAdapterIdentifiers & string[];
+  reconfigure?: TReconfigureAdapter;
+  status?: TAdapterStatus;
+};
+
+const defaultProps: Pick<TProps, 'recondigure' | 'status'> = {
```

Comment:
```suggestion
const defaultProps: Pick<TProps, 'reconfigure' | 'status'> = {
```

Reply:
Also, shouldn't this be `adapterIdentifiers`? 🤔

```suggestion
const defaultProps: Pick<TProps, 'adapterIdentifiers' | 'status'> = {
```

Reply:
Yes, thanks. Noticed that too :)